### PR TITLE
mdbookでGitHub Pagesに公開

### PR DESCRIPTION
**Pull request by sksat**
## Overview
mdbookでMarkdownをHTMLに変換してGitHub Pagesで公開する

- [x] mdbookでビルド
- [x] `SUMMARY.md`を適切な形にする
- [ ] 既存のMarkdownをmdbook向けに少し整える
- [x] GitHub Pagesにデプロイ

## Issue
- #3 

## Details
数式が`$``$`で記述されているが，mdbookはMathJaxでよくある`$$`をサポートしていない．
  - https://rust-lang.github.io/mdBook/format/mathjax.html

そのため，mdbookでサポートされている`\\(\\)`にビルド前に変換するスクリプト(`./prepro.sh`)を追加した．

##  Validation results
Link to tests or validation results.

## Scope of influence
eg. The behavior of XX will be change.

## Supplement
Write additional comments if you need.

## Note
- If there are related Projects, tie them together.
- Assignees should be set if possible.
- Reviewers should be set if possible.
- Set `priority` label if possible.

### CODE REVIEW

**Comment by sksat** [comment]
`SUMMARY.md`をいいかんじにするついでにrebase

**Comment by meltingrabbit** [comment]
image


やった．

@sksat rootでぃディレクトリでいい？

**Comment by meltingrabbit** [comment]
いい感じじゃん．

**Comment by sksat** [comment]
オ、よさそう

**Comment by meltingrabbit** [comment]
https://ut-issl.github.io/s2e-documents/Tutorials/HowToAddControlAlgorithms.html

ちょいちょい画像が死んでる？

**Comment by meltingrabbit** [comment]
- https://github.com/ut-issl/s2e-documents/blob/gh-pages/Tutorials/figs/ControlAlgorithm_ComponentControl_result5.png
- https://github.com/ut-issl/s2e-documents/blob/main/Tutorials/figs/ControlAlgorithm_ComponentControl_result5.png


謎の画像の突然死．

**Comment by meltingrabbit** [comment]
sedのせい？

**Comment by sksat** [comment]
これそもそもcommitされてる画像が死んでるのか．なぜ......

**Comment by sksat** [comment]
あこれ`find -type f`しかしてないから`sed`が突き刺さってるのか．そりゃおかしくなる．

**Comment by sksat** [comment]
よいのでは

**Comment by 200km** [comment]
ありがとうございます！ざっとみてみてほぼうまく行っていると思うのですが、次の部分が表示がおかしいようです。

- [GeoPotential](https://ut-issl.github.io/s2e-documents/Specifications/Disturbance/Spec_GeoPotential.html)　
  - 数式が一部おかしい
- [SurfaceForce](https://ut-issl.github.io/s2e-documents/Specifications/Disturbance/Spec_SurfaceForce.html)
  - 数式が一部おかしい
- [AirDrag](https://ut-issl.github.io/s2e-documents/Specifications/Disturbance/Spec_SurfaceForce_AirDrag.html)
  - 表も崩れている？
- [Atmosphere](https://ut-issl.github.io/s2e-documents/Specifications/Environment/Spec_Atmosphere.html)
  - インデントが崩れてる？
- GNSS Satellites (Japanese)などのJapaneseの方のリンクが無くなっている

**Comment by 200km** [comment]
「既存のMarkdownをmdbook向けに少し整える」の部分、数が多ければ(確認も含めて)手分けしてやるので、修正方針教えていただければ、こちら側で割り振ります。

**Comment by meltingrabbit** [comment]
そういえば，mathjaxは `\bm` が使えないので， `\boldsymbol` とかに置換する必要はありそうですね．

**Comment by sksat** [comment]
これはやってみて表示が崩れてるとかがあれば整えた方がいいかなと思っていたぐらいなので，そんなに修正点無いかもです．指摘してもらったインデントがおかしい，みたいなやつですかね． > 既存のMarkdownをmdbook向けに少し整える

**Comment by sksat** [comment]
MathJaxは前提としてよいと思うので，そういうのは`prepro.sh`ではなくMarkdownの方を直すかんじでいきたいですね． > `\bm`

**Comment by 200km** [comment]
分かりました。`\bm`や表のズレはMarkdown修正するという感じでいいですか？  
研究室の人に `good first issue`でやってもらおうかなとも思っています。

**Comment by sksat** [comment]
なるほど．それでいいと思います．そうしたらこのPRは一旦これで大丈夫ですかね． > Markdown修正．

**Comment by 200km** [comment]
@sksat 先程軽く話したとおり、一旦これでマージしていただいて良いと思います。  
@meltingrabbit 良いでしょうか？

**Comment by meltingrabbit** [comment]
いいと思います．

**Review by meltingrabbit — APPROVED** [review]
